pkg/policy/policies_iterator.go

```diff
@@ -0,0 +1,54 @@
+package policy
+
+import (
+	"github.com/aquasecurity/tracee/pkg/utils"
+)
+
+// PoliciesIterator is an iterator for Policies.
+type PoliciesIterator struct {
+	policies []*Policy
+	index    int
+}
+
+// HasNext returns true if there are more policies to iterate.
+func (i *PoliciesIterator) HasNext() bool {
+	return i.index < len(i.policies)
+}
+
+// GetNext returns the next policy in the iteration.
+func (i *PoliciesIterator) GetNext() *Policy {
```

Comment:
Returning a pointer is not the ideal result, but keep in mind that Policy will become a read-only interface in the future, so this will be resolved.